fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -76,6 +73,13 @@ public SendForsendelseMedId mapFrom(EDUCore eduCore, X509Certificate certificate
             senderRef = eduCore.getId();
         } else {
             senderRef = eduCore.getSender().getRef();
+            log.debug("sender.ref={}, validating", senderRef);
+            try {
+                UUID.fromString(senderRef);
+            } catch (IllegalArgumentException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)